tsdb/docs/format/wal.md

```diff
@@ -85,4 +85,34 @@ and specify an interval for which samples of a series got deleted.
 │ └─────────┴───────────────────┴───────────────────┘ │
 │                        . . .                        │
 └─────────────────────────────────────────────────────┘
+<<<<<<< HEAD
+=======
 ```
+
+### Exemplars records
+
+Exemplars records encode exemplars as a list of quadruples `(series_id, timestamp, max_time, exemplar)`.
+Series reference and timestamp are encoded as deltas w.r.t the first sample.
```

Comment:
And this sentence should come after the next one about the first row to make more sense.
```suggestion
Series reference and timestamp are encoded as deltas w.r.t the first row.
```